AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/commands/teleport/TpBlockCommand.java

```diff
@@ -31,7 +31,11 @@ public boolean onCommand(
         if (args.length == 0) {
             if (atPlayer instanceof ATFloodgatePlayer
                     && MainConfig.get().USE_FLOODGATE_FORMS.get()) {
-                ((ATFloodgatePlayer) atPlayer).sendBlockForm();
+                if (!((ATFloodgatePlayer) atPlayer).getVisiblePlayerNames().isEmpty()) {
+                    ((ATFloodgatePlayer) atPlayer).sendBlockForm();
+                } else {
+                    CustomMessages.sendMessage(sender, "Error.noOthersToTP");
```

Comment:
Instead of noOthersToTP, I would specify noPlayerInput, since /tpblock shouldn't need the target player online to block them (unless I've done something horribly wrong)